vllm/engine/arg_utils.py

```diff
@@ -892,6 +893,9 @@ def create_engine_config(self, ) -> EngineConfig:
             embedding_mode=model_config.embedding_mode,
             preemption_mode=self.preemption_mode,
             num_scheduler_steps=self.num_scheduler_steps,
+            _send_delta_data=(envs.VLLM_USE_RAY_SPMD_WORKER
+                              and envs.VLLM_SPMD_SEND_DELTA_DATA
```

Comment:
if sending data always makes sense in ray spmd worker, we don't need this flag.

Reply:
this was from the feedback to make a guard for debugging purpose. https://github.com/vllm-project/vllm/pull/7109#issuecomment-2284998715

If you want to remove it, I can just do that as well